CSS/README.md

```diff
@@ -185,15 +189,74 @@
+  >   - **rem:** Relative to font-size of the root element (which is the html element in HTML documents)
+  >  - C) Viewport Percentage Lengths
+  >    - **vw:** (viewport width) This is the “viewport width” unit. 1vw is equal to 1/100th of the width of the viewport. For example, if the width of the window is 1000px, 1vw will be 10px.
+  >    - **vh:** (viewport height) Same as the vw (viewport width) unit only it is based on the viewport height instead. 1vh is equal to 1/100th of the viewport height. For example, if the browser’s height is 900px, 1vh would evaluate to 9px.
+  >    - **vmin:** vmin equals to 1/100th of the minimum value between the height and the width of the viewport. In other words, 1/100th of the side with the smallest length. For example, if the dimensions were 1200 x 800, the value would be 8px.
+  >    - **vmax:** Similar to vmin, except it equals to 1/100th of the maximum value between the height and the width of the viewport. In other words, 1/100th of the side with the largest length. For example, if the dimensions were 1200 x 800, the value would be 12px.
+
+- [x] 32. Overflow
+  
+  > - Every single element on a page is a rectangular box. If you don't set the height of a box, the height of that box will grow as large as it needs to be to accommodate the content. But what happens when you do set a specific height or width on a box, and the content inside cannot fit? That is where the CSS overflow property comes in, allowing you to specify how you would like that handled.
+  > - There are four values for the overflow property: `visible` (default), `hidden`, `scroll`, and `auto`. There are also sister properties `overflow-y` and `overflow-x`, which enjoy less widespread adoption.
+
+- [x] 33. Pseudo classes
+
+  > - A CSS pseudo-class is a keyword added to a selector that specifies a special state of the selected element(s). For example, `:hover` can be used to change a button's color when the user hovers over it. Few examples shown below (Full Reference - https://css-tricks.com/pseudo-class-selectors/)
+  > - Link related
+  >   - `:link` this is essentially the same as a[href].
+  >   - `:visited` already been visited by the current browser.
+  >   - `:hover` mouse cursor rolls over a link, it's hover state.
+  >   - `:active` being clicked on or otherwise activated.
+  > - Input
+  >   - `:focus` puts cursor on the input field thus making it ready to enter data
+  > - Position Based
+  >   - `:first-child` Selects the first element within a parent
+  >   - `:last-child` Selects the last element within a parent
+  >   - `:nth-child()` Selects elements based on a simple provided algebraic expression (e.g. "2n" or "4n-1")
+
+- [x] 34. CSS Selectors
+  
+  > - CSS selectors define the elements to which a set of CSS rules apply.
+  > - **Basic Selectors**
+  >    - Type selector (`input`)
```

Comment:
`element selector` instead of `type selector` ?

Reply:
make sense, changed to element selector